### ISSUE TITLE
Core training skipped, no valid domain files found

### ISSUE THREAD

**Issue by SuvenduDash** [RasaHQ/rasa#4827]
Rasa version:
Rasa 1.4.6

Rasa X version (if used & relevant):

Python version:
3.6.8

Operating system (windows, osx, ...):
Ubuntu

Issue:
Core training skipped, no valid domain files found. I have attached my stories.md and nlu.md
I am running a rasa train command but the core training is being skipped. My stories include a menu path that loops the main line stories as well.

Error (including full traceback):

`rasa train
2019-11-23 13:20:37 ERROR    pykwalify.core  - validation.invalid
2019-11-23 13:20:37 ERROR    pykwalify.core  -  --- All found errors ---
2019-11-23 13:20:37 ERROR    pykwalify.core  - ["Value 'None' is not a dict. Value path: '/slots'"]
2019-11-23 13:20:37 WARNING  rasa.importers.rasa  - Loading domain from 'domain.yml' failed. Using empty domain. Error: 'Failed to validate yaml file. Please make sure the file is correct and all mandatory parameters are specified; to do so, take a look at the errors logged during validation previous to this exception.'
Training NLU model...
2019-11-23 13:20:57 INFO     rasa.nlu.training_data.training_data  - Training data stats: 
	- intent examples: 161 (6 distinct intents)
	- Found intents: 'services', 'location', 'contact', 'about', 'goodbye', 'greet'
	- Number of response examples: 0 (0 distinct response)
	- entity examples: 26 (2 distinct entities)
	- found entities: '            type_of_services', 'type_of_services'

/home/ocpl/Documents/environments/venv/lib/python3.6/site-packages/rasa/nlu/training_data/training_data.py:340: UserWarning: Entity '            type_of_services' has only 1 training examples! minimum is 2, training may fail.
  "".format(entity_type, count, self.MIN_EXAMPLES_PER_ENTITY)
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Starting to train component WhitespaceTokenizer
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Finished training component.
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Starting to train component RegexFeaturizer
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Finished training component.
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Starting to train component CRFEntityExtractor
2019-11-23 13:20:57 WARNING  rasa.nlu.extractors.crf_entity_extractor  - Misaligned entity annotation for 'develomentWEB DEVELOPMENT' in sentence 'are you providing web develomentWEB DEVELOPMENT service.' with intent 'services'. Make sure the start and end values of the annotated training examples end at token boundaries (e.g. don't include trailing whitespaces or punctuation).
2019-11-23 13:20:57 WARNING  rasa.nlu.extractors.crf_entity_extractor  - Misaligned entity annotation for 'wantDIGITAL MARKETING' in sentence 'I wantDIGITAL MARKETING service .' with intent 'services'. Make sure the start and end values of the annotated training examples end at token boundaries (e.g. don't include trailing whitespaces or punctuation).
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Finished training component.
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Starting to train component EntitySynonymMapper
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Finished training component.
2019-11-23 13:20:57 INFO     rasa.nlu.model  - Starting to train component CountVectorsFeaturizer
2019-11-23 13:20:58 INFO     rasa.nlu.model  - Finished training component.
2019-11-23 13:20:58 INFO     rasa.nlu.model  - Starting to train component CountVectorsFeaturizer
2019-11-23 13:20:58 INFO     rasa.nlu.model  - Finished training component.
2019-11-23 13:20:58 INFO     rasa.nlu.model  - Starting to train component EmbeddingIntentClassifier
2019-11-23 13:21:02.466170: E tensorflow/stream_executor/cuda/cuda_driver.cc:318] failed call to cuInit: UNKNOWN ERROR (303)
Epochs: 100%|███████████████████████████████████████████████████████████████████████████| 300/300 [00:29<00:00, 12.07it/s, loss=0.531, acc=0.994]
2019-11-23 13:21:32 INFO     rasa.utils.train_utils  - Finished training embedding policy, train loss=0.531, train accuracy=0.994
2019-11-23 13:21:32 INFO     rasa.nlu.model  - Finished training component.
2019-11-23 13:21:34 INFO     rasa.nlu.model  - Successfully saved model into '/tmp/tmpgoro_39m/nlu'
NLU model training completed.
2019-11-23 13:21:34 ERROR    pykwalify.core  - validation.invalid
2019-11-23 13:21:34 ERROR    pykwalify.core  -  --- All found errors ---
2019-11-23 13:21:34 ERROR    pykwalify.core  - ["Value 'None' is not a dict. Value path: '/slots'"]
2019-11-23 13:21:34 WARNING  rasa.importers.rasa  - Loading domain from 'domain.yml' failed. Using empty domain. Error: 'Failed to validate yaml file. Please make sure the file is correct and all mandatory parameters are specified; to do so, take a look at the errors logged during validation previous to this exception.'
2019-11-23 13:21:34 ERROR    pykwalify.core  - validation.invalid
2019-11-23 13:21:34 ERROR    pykwalify.core  -  --- All found errors ---
2019-11-23 13:21:34 ERROR    pykwalify.core  - ["Value 'None' is not a dict. Value path: '/slots'"]
2019-11-23 13:21:34 WARNING  rasa.importers.rasa  - Loading domain from 'domain.yml' failed. Using empty domain. Error: 'Failed to validate yaml file. Please make sure the file is correct and all mandatory parameters are specified; to do so, take a look at the errors logged during validation previous to this exception.'
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'greet' on line 2. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'about' on line 5. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'location' on line 8. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'contact' on line 11. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'goodbye' on line 15. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'greet' on line 19. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'goodbye' on line 22. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'greet' on line 26. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'goodbye' on line 30. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'about' on line 35. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'location' on line 39. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'contact' on line 43. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 48. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 51. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 58. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 61. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 68. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 71. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 78. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 81. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 88. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 91. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 98. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 101. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 111. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 120. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 129. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 138. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 147. Please, make sure that all intents are listed in your domain yaml.
2019-11-23 13:21:34 WARNING  rasa.core.training.dsl  - Found unknown intent 'services' on line 156. Please, make sure that all intents are listed in your domain yaml.
Your Rasa model is trained and saved at '/home/ocpl/Desktop/Oretes_ChatBot/Oretes Chat bot/models/nlu-20191123-132134.tar.gz'.
Core training was skipped because no valid domain file was found. Only an nlu-model was created.Please specify a valid domain using '--domain' argument or check if the provided domain file exists.
`
**Command or request that led to error:**
`rasa train`
**Content of domain file (domain.yml) (if relevant):**

`intents:
  - greet
  - goodbye
  - contact
  - about
  - location
  - services

slots:
type_of_services:
    type: categorical
    values:
    - SAP ISU
    - WEB DEVELOPMENT
    - MOBILE APPLICATION DEVELOPMENT 
    - DIGITAL MARKETING
    - TRAINING AND MENTORING
    - RESEARCH CONSULTANCY 

entities:
- type_of_services

templates:
  utter_greet:
  - text: "Hello there, Welcome to Ores . How can i help you ?"
  
  utter_goodbye:
  - text: "Thank you, I am happy to help you. For more details about us please visit our website:http://ocpl.com "

  utter_about:
  - text: "Ocpl  is provides technology services based on Future Ready Programs (SAP Leonardo, AI, Block Chain, IOT) and Digital Marketing which addresses complex business needs.For more info visit  http://ocpl.com "

  
  utter_location:
  - text: "OCPL Pvt Ltd is located in Bhubaneswar,India.Folllow us at google maps https://g.page/ocpl-ltd "

  utter_contact:
  - text: "For contact details, follow us at http://ocpl.com/contact-us/"
  utter_about_services:
  - text: "We offer all type of services.For more details about services please visit our website:http://ocpl.com/services"

actions:
- utter_greet
- utter_goodbye
- utter_location
- utter_contact
- utter_about
- utter_about_services`

**Comment by sara-tagger** [comment]
Thanks for the question, our Github issue board is for **bugs and feature requests only,** though 😕
Please ask this question in the [forum✨](https://forum.rasa.com/c/rasa)

###### If you were describing a bug, please create a [report](https://github.com/RasaHQ/rasa/issues/new?labels=bug&template=bug_report.md) and fill out the template.
